gatsby/content/blog/2022/07/2022-08-03-synapse-1.64.mdx

```diff
@@ -0,0 +1,98 @@
+---
+date: '2022-08-03'
+title: Synapse 1.64 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+It's that time again: there's a new Synapse release, fresh out of the oven!
+Let's take a look at what's inside [Synapse
+1.64](https://github.com/matrix-org/synapse/releases/tag/v1.64.0).
+
+## Delegating email verification is now deprecated
+
+[Synapse 1.4.0](https://github.com/matrix-org/synapse/releases/tag/v1.4.0)
+introduced a configuration option (`account_threepid_delegates.email`) to allow
+homeservers to delegate validating the ownership of email addresses to an
+external identity server. This validation is used by Synapse when associating
+email addresses to a Matrix account, or before performing a password reset.
+
+As of Synapse 1.64, this option is deprecated. This is because this option
+relies on old API endpoints that have since been removed from the Matrix
+specification. Server administrators who still need the ability to validate the
+ownership of email addresses must configure an SMTP server in their Synapse
+configuration file. See the [configuration
+guide](https://matrix-org.github.io/synapse/v1.64/usage/configuration/config_documentation.html#email)
+for more information.
+
+We plan to fully remove this configuration option in Synapse 1.66, which is
+expected to be released in 4 weeks.
+
+Note that the equivalent option to validate the ownership of phone numbers
+(`account_threepid_delegates.msisdn`) can still be used, though we expect to
+deprecate it in a future release of Synapse.
+
+## Improved TLS support for sending emails
+
+When configuring an SMTP server to use to send out emails to users, server
+administrators can configure Synapse to use TLS to communicate to that server.
+Until now, only [STARTTLS](https://en.wikipedia.org/wiki/Opportunistic_TLS) was
+supported in this case, and a long-standing pain point for server administrators
+has been the lack of support for implicit TLS (i.e. the use of TLS without a
+STARTTLS upgrade).
+
+Synapse 1.64 introduces support for implicit TLS. As of this version of Synapse,
+server administrators can set the new configuration option `force_tls` to `true`
+in the `email` section of their configuration file.
+
+See the [configuration
+guide](https://matrix-org.github.io/synapse/v1.64/usage/configuration/config_documentation.html#email)
+for more information. Note that the `require_transport_security` setting of the
+`email` section of the configuration file is ignored when implicit TLS is
+enabled.
+
+## Everything else
+
+This version of Synapse introduces support for room version 10! This new room
+version enables support for the new `knock_restricted` join rule, to allow
+knocking into rooms which are otherwise restricted to members of a specific room
+(or space). See [the Matrix specification about room version
+10](https://spec.matrix.org/v1.3/rooms/v10/) for more information.
+
+Synapse 1.64 features a new rate limiter to limit the rate of joins to the same
```

Comment:
This wording makes it sound like we are talking about a *different* version. I know it's difficult to find ways to avoid starting each section with "This version also"... how about:

```suggestion
Additionally, Synapse 1.64 features a new rate limiter to limit the rate of joins to the same
```